Products/ZenModel/RRDDataPoint.py

```diff
@@ -199,6 +216,95 @@ def isCounter(self):
         # rrdmin is defined as a string above
         return self.rrdtype == 'COUNTER' or (self.rrdtype =='DERIVE' and str(self.rrdmin) == '0')
 
+    def getTags(self, context):
+        """Return dict of evaluated tags for given context.
+
+        This includes tags that are the result of setting the store, forward,
+        and other properties that should influence the metric tags for this
+        datapoint.
+
+        """
+        tags = {}
+
+        if self.store is False:
+            tags[NO_STORE_KEY] = TRUTH_STRING
```

Comment:
Just to be clear, the metric-consumer impl just looks for the presence of the tag key, the value is never looked at.  It will work with this, but won't work if someone adds `no-store=False`

Reply:
Yes, I agree that entering "no-store:False" or something like that into the extra tags will result in it being stored, but I don't think that's likely to happen since there's a checkbox right about it saying "Store in CZ" that handles that without doing extra tags at all.